packages/apps/documentation/src/pages/styles/icons.md

```diff
@@ -337,6 +337,880 @@
   </div>
 </div>
 
+### Legacy Icons
+
+<div class="row">
+  <div class="oui-icon-badge">
+    <div class="oui-icon-badge__preview"><span class="oui-icon oui-icon-legacy-SMSEmitting" aria-hidden="true"></span></div>
```

Comment:
Should be `oui-icon-legacy-sms-emitting` instead.
This must come from the filename of the svg file.